Svc/Deframer/docs/sdd.md

```diff
@@ -386,27 +392,103 @@ Vertical dashed lines represent component code.
 Solid horizontal arrows represent synchronous port invocations, and open
 horizontal arrows represent asynchronous port invocations.
 
-![Active byte stream driver, command packet](img/sequence-diagrams/active-cmd-packet.png)
+```mermaid
+sequenceDiagram
+    activate activeComm
+    activeComm->>buffMgr: Allocate frame buffer FB
+    buffMgr-->>activeComm: Return FB
+    activeComm->>activeComm: Fill FB with framed data
+    activeComm->>deframer: Send FB[framedIn]
+    deframer->>buffMgr: Allocate packet buffer PB [bufferAllocate]
+    buffMgr-->>deframer: Return PB
+    deframer->>deframer: Deframe FB into PB
+    deframer->>deframer: Copy PB into a command packet C
+    deframer-)cmdDisp: Send C [comOut]
+    deframer->>buffMgr: Deallocation PB [bufferDeallocate]
+    buffMgr-->>deframer: 
+    deframer->>buffMgr: Deallocation FB [framedDeallocate]
+    buffMgr-->>deframer: 
+    deframer-->>activeComm: 
+    deactivate  activeComm 
+    activate cmdDisp
+    cmdDisp->>deframer: Send cmd response [cmdResponseIn]
+    deframer-->>cmdDisp: 
+    deactivate cmdDisp
+```
 
 **Sending a file packet:**
 The following sequence diagram shows what happens when `activeComm`
 sends data to `deframer`, and `deframer` decodes the data into a file packet.
 
-![Active byte stream driver, file packet](img/sequence-diagrams/active-file-packet.png)
+```mermaid
+sequenceDiagram
+    activate activeComm
+    activeComm->>buffMgr: Allocate frame buffer FB
+    buffMgr-->>activeComm: Return FB
+    activeComm->>activeComm: Fill FB with framed data
+    activeComm->>deframer: Send FB [framedIn]
+    deframer->>buffMgr: Allocate packet buffer PB [bufferAllocate]
+    buffMgr-->>deframer: Return PB
+    deframer->>deframer: Deframe FB into PB
+    deframer-)fileUplink: Send PB [bufferOut]
+    deframer->>buffMgr: Deallocate FB [framedDeallocate]
+    buffMgr-->>deframer: 
+    deframer-->>activeComm: 
+    deactivate activeComm
+    activate fileUplink
+    fileUplink->>-buffMgr: Deallocate PB
```

Comment:
Is this a typo? `>>` between `-` and `-`.

Reply:
The `-` is a shortcut to deactivate a thread. But the code contained an error, I deactivated the component at the wrong time.